tests/kafkatest/services/kafka/kafka.py

```diff
@@ -368,7 +368,7 @@ def start_node(self, node, timeout_sec=60):
     def pids(self, node):
         """Return process ids associated with running processes on the given node."""
         try:
-            cmd = "jcmd | grep -e %s -e %s | awk '{print $1}'" % (self.java_class_name(), self.apache_java_class_name())
+            cmd = "jcmd | grep -e %s -e %s | awk '{print $1}'" % self.java_class_name()
```

Comment:
This should be ( self.java_class_name(), self.java_class_name() ) since there are two %s

Reply:
Yeah, I meant to remove one of the %s, good catch.

Reply:
Fixed.